test/Microsoft.AspNet.Mvc.ViewFeatures.Test/Rendering/HtmlHelperPasswordTest.cs

```diff
@@ -345,6 +345,60 @@ public static TheoryData PasswordFor_WithComplexExpressionsData
             Assert.Equal(expected, HtmlContentUtilities.HtmlContentToString(result));
         }
 
+        [Fact]
+        public void Password_UsesSpecifiedExpressionButIgnoresValue()
```

Comment:
`Password_UsesSpecifiedExpressionForNames_IgnoresExpressionValue()`. Same for similar test below.